my_code.py

```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import matplotlib
+import os
+from astropy.io import fits
+import glob
+data = fits.getdata('/Users/bostroem/Desktop/images/hubble_img.fits')
+col1 = data[20, :] 
+col2 = data[200, :]
+col3 = data[800, :]
+pyplot.figure()
+pyplot.plot(np.arange(1024), col1)
+mean = np.mean(col1)
+pyplot.axhline(mean)
+pyplot.legend(['Column 20', 'Mean'])
+pyplot.title('Plot of Column 20 Values and Mean')
+pyplot.xlabel('Pixels')
+pyplot.ylabel('Intensity')
+pyplot.figure()
+pyplot.plot(np.arange(1024), col2)
```

Comment:
can just do
`pyplot.plot(col2)` and it will automatically use index values on x-axis

(``np.arange`` is unnecessary)